tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_pod_disruption_budget.tpl

```diff
@@ -0,0 +1,75 @@
+{{ define "stress-test-addons.pod-disruption-budget" }}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: disruption-budget-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: {{ .Release.Name }}
+spec:
+  # Jobs do not implement `scale` otherwise we could set `minAvailable: 100%` or `maxUnavailable: 0` instead.
+  # Work around this by setting `minAvailable` to a number that will never be reached to simulate 100%
+  # so that the disruption budget will work in parallel pod scenarios (completionMode: indexed)
+  minAvailable: 10000
```

Comment:
this also prevents watcher from deleting the pod right?

Reply:
No, direct pod deletions are still allowed, it's node evictions that are prevented.